lib/helpers.js

```diff
@@ -57,9 +57,10 @@ exports.prefixedHash = x => {
 
 exports.sha256 = x => crypto.createHash('sha256').update(x).digest('base64');
 
-exports.generateNonce = async ({entryStyle = 'human'} = {}) => {
+exports.generateNonce = async ({entryStyle = 'human', nonceLength = 9}
+= {}) => {
```

Comment:
Usual style used elsewhere is this:
```suggestion
exports.generateNonce = async ({
  entryStyle = 'human', nonceLength = 9
} = {}) => {
```

Reply:
Updated, thanks!